src/frida/tracer.py

```diff
@@ -437,8 +438,12 @@ def objc_arg(m):
             varargs = False
             try:
                 with open(os.devnull, 'w') as devnull:
-                    output = subprocess.check_output(["man", "-P", "col -b", "2", function.name], stderr=devnull)
-                match = re.search(r"^SYNOPSIS(?:.|\n)*?((?:^.+$\n)* {5}" + function.name + r"\(.*\n(^.+$\n)*)(?:.|\n)*^DESCRIPTION", output.decode(), re.MULTILINE)
+                    output_argv=["man"]
+                    if platform.system() != "Darwin":
+                        output_argv.extend(["-E","UTF-8"])
+                    output_argv.extend(["-P", "col -b", "2", function.name])
+                    output = subprocess.check_output(output_argv, stderr=devnull)
+                match = re.search(r"^SYNOPSIS(?:.|\n)*?((?:^.+$\n)* {5}" + function.name + r"\(.*\n(^.+$\n)*)(?:.|\n)*^DESCRIPTION", output.decode("UTF-8",errors="replace"), re.MULTILINE)
```

Comment:
Space after comma, and there's also a convention that's a bit more subtle: if a string value is representing an enumeration value, like "UTF-8" and "replace" are here, they should be in single quotes. So double-quotes for free-form strings, and single-quotes for enumeration values (if only python had atoms/symbols :) ).